### PR TITLE
show-error-label-when-server-is-unresponsive

### PR DESCRIPTION
**Pull request by oanaBiancaSchiopu**
image

image

### CODE REVIEW

File: rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/RMController.java [hunk 1]
```diff
@@ -572,6 +572,11 @@ public void onFailure(Throwable caught) {
                 } else {
                     error("Failed to fetch RM State: " + JSONUtils.getJsonErrorMessage(caught));
                 }
+                if (caught.getMessage().contains("HTTP 401 Unauthorized")) {
+                    rmPage.showErrorLabel("You have been disconnected from the server. Reload the page and log into the portal.");
```
Comment (fviale) [inline rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/RMController.java:576]: You have been disconnected from the server. Reload the page in order to log into the portal again.

**Comment by fviale** [comment]
@oanaBiancaSchiopu can you add some screenshots ?

**Comment by oanaBiancaSchiopu** [comment]
image

**Comment by oanaBiancaSchiopu** [comment]
image

**Comment by fviale** [comment]
@oanaBiancaSchiopu looks good, I would add some space between the Network Error button and the message

**Review by fviale — APPROVED** [review]

**Pull request merged**